src/plugins/workspace/server/plugin.ts

```diff
@@ -26,18 +26,28 @@ import { IWorkspaceDBImpl } from './types';
 import { WorkspaceClientWithSavedObject } from './workspace_client';
 import { WorkspaceSavedObjectsClientWrapper } from './saved_objects';
 import { registerRoutes } from './routes';
-import { ConfigSchema } from '../config';
+import { ConfigSchema, FEATURE_FLAG_KEY_IN_UI_SETTING } from '../config';
 
 export class WorkspacePlugin implements Plugin<{}, {}> {
   private readonly logger: Logger;
   private client?: IWorkspaceDBImpl;
+  private coreStart?: CoreStart;
   private config$: Observable<ConfigSchema>;
+  private enabled$: BehaviorSubject<boolean> = new BehaviorSubject(false);
+  private loopRequestTimer?: NodeJS.Timeout;
```

Comment:
Was this used anywhere?

Reply:
Oh, my fault. There was a timer to check the flag in kibana index and try to keep the enabled$ consistent with the real flag in kibana index. Done for removing this.